assets/js/map.js

```diff
@@ -109,8 +112,8 @@ const MapBase = {
       zoomControl: false,
       crs: L.CRS.Simple,
       layers: [mapLayers[this.themeOverride || Settings.baseLayer]],
-      zoomSnap: false,
-      zoomDelta: (this.maxZoom - this.minZoom) / ((this.maxZoom - this.minZoom) * 2),
+      zoomSnap: 0,
+      zoomDelta: 0.5,
       wheelPxPerZoomLevel: 140,
       wheelDebounceTime: 150,
```

Comment:
Lets unify that zoom with Collectors map:

```
zoomSnap: false,
zoomDelta: (this.maxZoom - this.minZoom) / ((this.maxZoom - this.minZoom) * 2),
wheelPxPerZoomLevel: 70,
```

Reply:
The type of zoomSnap is supposed to be `Number`. I got confused with the next option think it was a typo, zoomDelta is assigned to 0.5 constantly.😮

Reply:
Oh, it was 
```javascript
(this.maxZoom - this.minZoom) / ((this.maxZoom - this.minZoom) * 2 - 1)
```
in historical commits in Collectors map.

Reply:
yes, you are right, zoomSnap should be typeof number. zoom delta always resolves to 0.5. My original idea about this setting was to make much more zoom levels on mobile devices, but zoomSnap: 0 solved that problem, i removed mobile part so i should change it back to 0.5 and i forgot.
Change only `wheelPxPerZoomLevel` that option changed between leaflet versions, and needs lower value in 1.9.4+v1.

Sorry for the confusion